superduperdb/components/component.py

```diff
@@ -62,7 +62,7 @@ def init(self):
                 item = item.unpack(db=self.db)
                 setattr(self, f.name, item)
 
-    @cached_property
```

Comment:
There will be a little problem using `cached_property` here
```
model.object = None
db.add(model)
model.object = lambda x: x + 1
# this will miss the object
db.add(model)
```